File: backend/review-service/src/main/java/it/roadies/review_service/data/dto/ReviewResponse.java
package it.roadies.review_service.data.dto;

import it.roadies.review_service.data.entity.ReviewType;

import java.time.LocalDateTime;
import java.util.UUID;

import lombok.Data;

@Data
public class ReviewResponse {
    private UUID id;
    private UUID travelId;
    private ReviewType reviewType;
    private String userId;
    private int rating;
    private String content;
    private LocalDateTime createdAt;
}

File: android-app/app/src/main/java/it/roadies/android_app/client/models/review/ReviewResponse.kt
package it.roadies.android_app.client.models.review

import java.time.LocalDateTime
import java.util.UUID

data class ReviewResponse(
    val id: UUID,
    val travelId: UUID,
    val reviewType: String,
    val userId: String,
    val rating: Int,
    val content: String,
    val createdAt: LocalDateTime,
    val reply: ReplyResponse?
)